apps/air-discount-scheme/web/next.config.js

```diff
@@ -45,6 +49,10 @@ module.exports = withSourceMaps(
         apiUrl: `${WEB_PUBLIC_URL}/api`,
         SENTRY_DSN,
         graphqlEndpoint: graphqlPath,
+        ddRumApplicationId: DD_RUM_APPLICATION_ID,
+        ddRumClientToken: DD_RUM_CLIENT_TOKEN,
```

Comment:
I'm assuming this is a secret .. not some sdk.
I believe we can rely on serverRuntimeConfig here instead of publicRuntimeConfig